<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALI</class>
 <widget class="qMRMLWidget" name="ALI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>950</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Inputs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_16">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Input type</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Input</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="SelectNodeLabel">
              <property name="text">
               <string>Select node</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ScanPathLabel">
              <property name="text">
               <string>Scan folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Models folder </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="InputTypeComboBox">
              <item>
               <property name="text">
                <string>CBCT - Work on Cone Beam CT scans </string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>IOS - Work on Intra Oral Scans</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="InputComboBox">
              <item>
               <property name="text">
                <string>File as input</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Folder as input</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="nodeTypes">
               <stringlist>
                <string>vtkMRMLVolumeNode</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditScanPath"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditModelPath"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="FillNodeLlabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SearchScanFolder">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SearchModelFolder">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="DownloadTestPushButton">
            <property name="text">
             <string>Download test file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DownloadModelPushButton">
            <property name="text">
             <string>Download latest models</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="optionsCollapsibleButton">
     <property name="text">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QVBoxLayout" name="OptionVLayout">
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Landmark selection :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Outputs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="SavePredictCheckBox">
          <property name="text">
           <string>Save prediction in scan folder</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Prediction ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SaveId">
            <property name="text">
             <string extracomment="pred">Pred</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="PredictFolderLabel">
              <property name="text">
               <string>Prediction folder</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLineEdit" name="SaveFolderLineEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="SearchSaveFolder">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Choose</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="advancedCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Input volume:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="qMRMLNodeComboBox" name="inputSelector">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Pick the input to the algorithm.</string>
            </property>
            <property name="nodeTypes">
             <stringlist>
              <string>vtkMRMLScalarVolumeNode</string>
             </stringlist>
            </property>
            <property name="showChildNodeTypes">
             <bool>false</bool>
            </property>
            <property name="addEnabled">
             <bool>false</bool>
            </property>
            <property name="removeEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Invert threshold: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="invertOutputCheckBox">
            <property name="toolTip">
             <string>If checked, values above threshold are set to 0. If unchecked, values below are set to 0.</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Thresholded volume:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="qMRMLNodeComboBox" name="outputSelector">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Pick the output to the algorithm.</string>
            </property>
            <property name="nodeTypes">
             <stringlist>
              <string>vtkMRMLScalarVolumeNode</string>
             </stringlist>
            </property>
            <property name="showChildNodeTypes">
             <bool>false</bool>
            </property>
            <property name="noneEnabled">
             <bool>true</bool>
            </property>
            <property name="addEnabled">
             <bool>true</bool>
            </property>
            <property name="removeEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Inverted volume:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="qMRMLNodeComboBox" name="invertedOutputSelector">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Result with inverted threshold will be written into this volume</string>
            </property>
            <property name="nodeTypes">
             <stringlist>
              <string>vtkMRMLScalarVolumeNode</string>
             </stringlist>
            </property>
            <property name="showChildNodeTypes">
             <bool>false</bool>
            </property>
            <property name="noneEnabled">
             <bool>true</bool>
            </property>
            <property name="addEnabled">
             <bool>true</bool>
            </property>
            <property name="removeEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="PrePredInfo">
     <property name="text">
      <string>Number of scans to process : 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="LandmarkNbrLabel">
     <property name="text">
      <string>Landmark 0 / X</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="LandmarkProgressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ScanNbrLabel">
     <property name="text">
      <string>Scan 0 / X</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="TotalrogressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PredictionButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Search landmarks</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ALI</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ALI</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>invertedOutputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ALI</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
